upgrade50Plus.sh

```diff
@@ -55,13 +59,62 @@ print_dd_progress() {
     done    
 } 
 
-
 check_preconditions() {
+    pretty "Starting upgrade..." "" "" "Starting upgrade..." ""
+    sleep 5
```

Comment:
Is there any reason for delaying the upgrade process for 10 secs?

Reply:
The idea was to let the user be able to read the messages on the S/Boled for a longer time. I thought the messages disappear to quickly.